packages/wallet/src/pages/transfer/pay.tsx

```diff
@@ -0,0 +1,81 @@
+import { AppLayout } from '@/components/layouts/AppLayout'
+import { Button } from '@/ui/Button'
+import { z } from 'zod'
+import Image from 'next/image'
+import { Form, useZodForm } from '@/ui/forms/Form'
+import { Input } from '@/ui/forms/Input'
+import { Select } from '@/ui/forms/Select'
+
+const paySchema = z.object({
+  fromAccount: z.string(),
+  incomingPaymentPointer: z.string(),
+  amount: z.number(),
+  currency: z.string()
+})
+
+export default function Pay() {
+  const form = useZodForm({
+    schema: paySchema
+  })
+
+  const handleSubmit = form.handleSubmit((data) => {
+    console.log(data)
+  })
+
+  return (
+    <AppLayout>
+      <div className="flex flex-col text-center lg:w-2/3">
```

Comment:
```suggestion
      <div className="flex flex-col lg:w-2/3">
```

Reply:
Hmmmm, we do need this, according to design, the Total balance and amount are centered.

Reply:
This will make every text in the `div` centered. We can put a wrapper around `Total Balance` and `Amount`.